openapi.yaml

```diff
@@ -16,175 +16,218 @@ host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/notes-api
 produces:
   - application/json
 paths:
-  /pets:
+  /notes:
     get:
-      summary: List all pets
+      summary: Get notes for a student
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - notes
+      operationId: getNotes
       parameters:
```

Comment:
Forgot to mention, we should add an authorization header parameter in every endpoint/method like [here](https://github.com/osu-mist/express-api-skeleton/blob/master/openapi.yaml#L60). This will allow SmartDoc to render the field in our developer portal.

Reply:
Sounds good, should be fixed now